packages/js/src/ui/components/primitives/Collapsible.tsx

```diff
@@ -1,32 +1,64 @@
-import { type Component, JSX, createEffect, createSignal } from 'solid-js';
+import { type Component, JSX, createEffect, createSignal, onCleanup } from 'solid-js';
 import { useStyle } from '../../helpers';
 
 type CollapsibleProps = JSX.IntrinsicElements['div'] & {
   class?: string;
   open: boolean;
 };
 
+const isInterpolateSizeSupported = () => {
+  return CSS.supports('interpolate-size', 'allow-keywords');
+};
```

Comment:
`interpolate-size` in the new CSS property that allows to transition on keywords for example from `0 to auto`